minimal/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "react-native-fullstack-boilerplate",
+  "version": "0.1.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "private": true,
+  "scripts": {
+    "start": "react-native-scripts start",
+    "exp:start": "exp start --lan",
+    "exp:android": "exp android",
+    "exp:ios": "exp ios",
+    "eject": "react-native-scripts eject",
+    "android": "react-native-scripts android",
+    "ios": "react-native-scripts ios",
+    "test": "node node_modules/jest/bin/jest.js --watch"
+  },
+  "jest": {
+    "preset": "jest-expo"
+  },
+  "devDependencies": {
+    "eslint": "4.18.1",
+    "eslint-config-airbnb": "16.1.0",
+    "eslint-config-airbnb-base": "12.1.0",
+    "eslint-plugin-import": "2.9.0",
+    "eslint-plugin-jsx-a11y": "6.0.3",
+    "eslint-plugin-react": "7.7.0",
+    "eslint-plugin-react-native": "3.2.1",
+    "jest-expo": "25.1.0",
+    "react-native-scripts": "1.11.1",
+    "react-test-renderer": "16.2.0"
+  },
+  "dependencies": {
+    "apollo-boost": "^0.1.4",
+    "expo": "^26.0.0",
+    "graphql": "^0.13.2",
+    "graphql-tag": "^2.8.0",
+    "react": "16.3.0-alpha.1",
+    "react-apollo": "^2.1.3",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-26.0.0.tar.gz"
```

Comment:
Version 27 would change this to: `"https://github.com/expo/react-native/archive/sdk-27.0.0.tar.gz"`